pallets/payment-streams/src/mock.rs

```diff
@@ -270,6 +270,10 @@ impl ExtBuilder {
         .assimilate_storage(&mut t)
         .unwrap();
 
+        crate::GenesisConfig::<Test> { current_price: 1 }
```

Comment:
What is this used for?

Reply:
I added this because the price of the dynamic-rate payment streams were initializing at 0 (since we don't have the equation for their price defined yet) and that was making testing more difficult than what it needed to be